src/kit/taosdemo/taosdemo.c

```diff
@@ -2382,9 +2483,19 @@ static void init_rand_data() {
     assert(g_rand_phase_buff);
     g_randdouble_buff = calloc(1, DOUBLE_BUFF_LEN * MAX_PREPARED_RAND);
     assert(g_randdouble_buff);
+    g_randuint_buff = calloc(1, INT_BUFF_LEN * MAX_PREPARED_RAND);
+    assert(g_randuint_buff);
+    g_randutinyint_buff = calloc(1, TINYINT_BUFF_LEN * MAX_PREPARED_RAND);
+    assert(g_randutinyint_buff);
+    g_randusmallint_buff = calloc(1, SMALLINT_BUFF_LEN * MAX_PREPARED_RAND);
+    assert(g_randusmallint_buff);
+    g_randubigint_buff = calloc(1, BIGINT_BUFF_LEN * MAX_PREPARED_RAND);
+    assert(g_randubigint_buff);
 
     for (int i = 0; i < MAX_PREPARED_RAND; i++) {
-        g_randint[i] = (int)(taosRandom() % 65535);
+        // TODO: make g_randint list include negative number
+        g_randint[i] = (int)(taosRandom() % 131069 - 65534);
```

Comment:
how about use "taosRandom() % RAND_MAX - (RAND_MAX >> 1) “  to extend max random value to all possible integer value?

Reply:
we can remove TODO after negative value generated

Reply:
sounds good, will make our data more real.